app/uk/gov/hmrc/pushpullnotificationsgateway/connectors/OutboundProxyConnector.scala

```diff
@@ -77,7 +77,7 @@ class OutboundProxyConnector @Inject()(appConfig: AppConfig,
     def failedRequestLogMessage(statusCode: Int) = s"Attempted request to ${notification.destinationUrl} responded with HTTP response code $statusCode"
     implicit val hc: HeaderCarrier =  HeaderCarrier()
     validateDestinationUrl(notification.destinationUrl) flatMap { validatedDestinationUrl =>
-      httpClient.POST[String, HttpResponse](validatedDestinationUrl, notification.payload, notification.forwardedHeaders.map(fh => (fh.key, fh.value)))
+      httpClient.POST[String, HttpResponse](validatedDestinationUrl, notification.payload)
```

Comment:
I think the request type being a string, it won't add the json content type automatically, unless you've tested this and it works somehow.

Reply:
Maybe it's easier to just add the content type explicitly as we won't really use the notification model for anything.

Reply:
That's true - I'll fix that